### PR TITLE
DMP-65 DMP-66 DMP-87

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
https://tools.hmcts.net/jira/browse/DMP-65
https://tools.hmcts.net/jira/browse/DMP-66
https://tools.hmcts.net/jira/browse/DMP-87

Transcriber templates added to config.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by davet1985** [comment]
Can you add a functional test for each notification template?

**Comment by cmagowan-hmcts** [comment]
> Can you add a functional test for each notification template?

done

File: src/functionalTest/java/uk/gov/hmcts/darts/notification/service/GovNotifyServiceTest.java [hunk 1]
```diff
@@ -20,19 +21,62 @@
...
+
+    @Test
+    void courtManagerApproveTranscript() throws NotificationClientException, TemplateNotFoundException {
+        String templateId = templateIdHelper.findTemplateId("court_manager_approve_transcript");
+        GovNotifyRequest govNotifyRequest = new GovNotifyRequest();
+        govNotifyRequest.setTemplateId(templateId);
+        govNotifyRequest.setEmailAddress(EMAIL_ADDRESS);
+        Map<String, String> parameterMap = new ConcurrentHashMap<>();
+        parameterMap.put("case_id", "TheCaseId");
+        govNotifyRequest.setParameterMap(parameterMap);
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/functionalTest/java/uk/gov/hmcts/darts/notification/service/GovNotifyServiceTest.java:42]: done

**Review by davet1985 — APPROVED** [review]
LGTM

**Pull request merged**